src/main/resources/application.yml

```diff
@@ -67,7 +67,7 @@ hiu:
     usingUnsecureSSL: ${Account_Service_Uses_Unsecure_SSL:true}
     url: ${Account_Service_Url}
     enable-offline-verification: ${ENABLE_OFFLINE_VERIFICATION:true}
-    hasBehindGateway: ${HAS_BEHIND_GATEWAY:false}
+    hasBehindGateway: ${HAS_BEHIND_GATEWAY:true}
```

Comment:
Let's not default it to true by default. Otherwise our local runs would start failing because we have not provided `clientId`, `clientSecret` etc.

Reply:
Done